index2.js

```diff
@@ -1,255 +0,0 @@
-const { exit } = require('process');
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-function getRandomInt(max) {
-    return Math.floor(Math.random() * Math.floor(max));
-  }
-
-start();
-
-async function start() {
-    console.log("Let's play a game where I (computer) make up a number (1-100) and you (human) try to guess it.")
-    let secretNumber = getRandomInt(100)
-
-    //console.log(secretNumber) // to see what computer guessed, develop delete when finished
-
-    let win= "Congrats! You guessed my number!! "
-
-
-//need to figure out how to create a loop or function for this ??
-
-
-    let firstGuess = await ask("What is your guess? ")
-    if(parseInt(firstGuess) === secretNumber){
-        console.log(win +"It took one try")
-        process.exit();
-    }
-    else if  (parseInt(firstGuess) < secretNumber )
-    {
-        console.log("Higher")
-    }
-    else 
-        {
-        console.log("Lower")        
-    }
-        
```

Comment:
We can shift things like this to be a `while` loop rather then `if..else` statements and then the user will be given as many tries to guess as needed rather then capping it at 16 because of it being hard coded